src/utils/components/CPUMemory/CPUMemory.tsx

```diff
@@ -19,17 +22,15 @@ const CPUMemory: FC<CPUMemoryProps> = ({ vm }) => {
   const { t } = useKubevirtTranslation();
   const itMatcher: V1InstancetypeMatcher = vm?.spec?.instancetype;
   const { instanceType, instanceTypeLoaded, instanceTypeLoadError } = useInstanceType(itMatcher);
+  const isVMRunning = vm?.status?.printableStatus === printableVMStatus.Running;
   const { vmi, vmiLoadError } = useVMI(getName(vm), getNamespace(vm));
 
-  if (vmiLoadError || (instanceTypeLoadError && !isEmpty(itMatcher)))
-    return <MutedTextSpan text={t('Not available')} />;
+  const cpu = vCPUCount(useCPUData(vm)) || instanceType?.spec?.cpu?.guest;
```

Comment:
u already made a custom hook useCpu.. why not getting instanceType there also and simply return the CPU? u dont need this ||

Reply:
also the vcpucount can be used there

Reply:
why not? because I think we don't need to put instanceType check everywhere. Really unnecessary. This is just a small refactoring, I am not changing anything in the logic, except that I am adding extra check to vmi check, in this PR.

Reply:
ur hook names useCPUData but it not retrieve, instanceType cpu data, if you already have a chance to improve this file and ur hook to retrieve also the instanceType cpu i think you should get from this hook the cpu and memory data instead of getting half and then in the component to check the other half.. just make more sense that you have 1 place to get the data